5-Heroku/server/todo_service.js

```diff
@@ -0,0 +1,94 @@
+const mongoose = require("mongoose");
+const uuid = require("uuid");
+require('dotenv').config();
+
+const Schema = mongoose.Schema;
+
+const MONGO_URI = process.env.MONGO_URI;
+
+mongoose.connect(MONGO_URI, {
+  useNewUrlParser: true,
+  useUnifiedTopology: true,
+});
+
+mongoose.connection.on("connected", () => {
+  console.log("MobgoDB is connected!");
+});
+
+const TodoSchema = new Schema({
+  user_id: String,
+  id: String,
+  text: String,
+  check: Boolean,
+});
+
+const Todo = mongoose.model("Todo", TodoSchema);
+
+function dbUpdateCallback(req, res, successStatus) {
+  return function (error, doc) {
+    if (error) {
+      console.log(error);
+      res.status(500).send("Database error!");
+    } else {
+      res.sendStatus(successStatus);
+    }
+  };
+}
+
+module.exports.getItems = (req, res) => {
```

Comment:
TodoService should be logic only it shouldn't know anything about requests / responses / etc..
this should be handled in the `controller` in your case the controller is your `index.js`

I would expect TodoService to implement a logical interface something like
```
class TodoService {
   getItems(userId) {
      //do some logic
      return items
   }
}
```